src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -52,95 +52,99 @@
 
 namespace
```

Comment:
Would we still need anonymous namespaces? Their point _is_ internal linkage, and if we can't use internal linkage because of Unity builds, they serve no real purpose, or do they?

Or would it make sense to use `namespace sf` as the outer one, at the low chance that another library uses similar names?

Reply:
We still need anonymous namespaces for non-unity builds. I think the suggestion of using `sf` is overkill, as the point of this PR is allowing SFML to be built as a unity build on its own, not mixed with other project files.

Reply:
> We still need anonymous namespaces for non-unity builds. 

Why? Yes, theoretically a symbol is accessible outside its translation unit, but by following the de-duplication rules for unity builds, we automatically ensure there are no conflicts, no?

Reply:
Anonymous namespaces have other benefits. One that comes to mind is that compilers will warn about unused internal-linkage functions in a `.cpp` file, while they will not warn otherwise as they cannot prove the function is not being called from anywhere else.